docs/toc.yml

```diff
@@ -148,3 +148,6 @@ entries:
 
 - file: user/FAQS
   title: FAQs
+
+- file: user/FlowVariables.md
```

Comment:
Remove the `.md` file extension and move the entry to be after user guide and before metrics. Do not forget to match the indentation.

Reply:
updated